apps/range-voting/app/components/Slider.js

```diff
@@ -52,7 +52,7 @@ class Slider extends React.Component {
   updateValueFromClientX(clientX) {
     const rect = this.getRect()
     const x = parseFloat(
-      Math.min(rect.width, Math.max(0, clientX - rect.x))
+      Math.floor(Math.min(rect.width, Math.max(0, clientX - rect.x)) * 1000 ) / 1000
```

Comment:
This will help with values with more than 4 decimal figures like 5.0005 that will be 5000.5 then floor to 5000 and become 5 after dividing again by 1000 but seem not really accurate to cover all cases